llvm/lib/IR/DiagnosticInfo.cpp

```diff
@@ -431,3 +431,5 @@ void DiagnosticInfoDontCall::print(DiagnosticPrinter &DP) const {
   if (!getNote().empty())
     DP << ": " << getNote();
 }
+
+void DiagnosticInfoSYCLWarning::print(DiagnosticPrinter &DP) const {}
```

Comment:
We should at least have a `FIXME: implement` here if we do not plan to implement this method immediately

Reply:
I am not sure that this method should be implemented